downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -68,8 +101,66 @@ To view bug fixes, see the [GitHub milestone for 2201.8.0 (Swan Lake)](https://g
 
 ### New features
 
+#### New Runtime Java APIs
+
+- Introduced the `getImpliedType()` API in the `io.ballerina.runtime.api.utils.TypeUtils` class to retrieve the referred type if a given type is a type-reference type or to retrieve the effective type if the given type is an intersection type.
+
+    ```java
+    Type getImpliedType(Type type)
+    ```
+
 ### Improvements
 
+#### Intersection type support in runtime Java APIs
+
+The following runtime APIs are now modified to return an intersection type rather than returning the effective type of the intersection.
```

Comment:
```suggestion
The following runtime APIs are now modified to return an intersection type rather than returning the effective type of the intersection.
```
Check why the word `return` is marked in red in this preview above.

Reply:
Seems a bug in git 😄